### PR TITLE
Rename all occurrences of `AddressBook` to `Realodex`

### PR DESCRIPTION
**Pull request by UdhayaShan1**
### Changes Made:

- Package Name Update: Changed the package name from seedu.address to seedu.realodex.
- Class Renaming:
- Renamed AddressBookParser to RealodexParser.
- Renamed AddressBook main class to Realodex.
- Storage Renaming:
- Renamed AddressBookStorage and JsonSerializableStorage to use the "Realodex" naming convention.
- Renamed the JSON storage file from addressbook.json to realodex.json.
- Gradle Build Update: Updated the build.gradle file.
- JSON Storage Renaming: Updated JSON-related packages and storage classes to reflect the new naming convention.
- Test Fixes: Fixed tests related to the realodex.json file and updated method names.
- Documentation Update: Updated teams' markdown files and remaining fields to reflect the changes.

### CODE REVIEW

File: src/main/java/seedu/realodex/commons/core/LogsCenter.java
Comment (4llysa) [inline src/main/java/seedu/realodex/commons/core/LogsCenter.java:1]: there are some isntances of ab3 in this file that needs editing

```src/main/java/seedu/realodex/commons/core/LogsCenter.java:78:     * Creates a logger named 'ab3', containing a {@code ConsoleHandler} and a {@code FileHandler}.```

```src/main/java/seedu/realodex/commons/core/LogsCenter.java:82:        baseLogger = Logger.getLogger("ab3");```

**Review by appleraincoat — APPROVED** [review]
Well done on this mammoth task, appreciate the effort!
Functionally the code is working now. Any missed instances of "addressbook" should be trivial or just in comments, which we can change on the fly as we proceed with the project.

**Review by jyztintan — APPROVED** [review]
thank you wery much sir! respect 🫡

**Review by wapisai — COMMENTED** [review]
slay lah!

**Comment by UdhayaShan1** [comment]
Nvm superseded by #129

**Pull request closed without merging**